addons/maintenance/tests/test_maintenance.py

```diff
@@ -97,3 +97,23 @@ def test_forever_maintenance_repeat_type(self):
         maintenance_request.with_context(default_stage_id=maintenance_stages[1].id).stage_id = done_maintenance_stage
         new_maintenance = self.env['maintenance.request'].search([('name', '=', 'Test forever maintenance'), ('stage_id', '=', maintenance_stages[0].id)])
         self.assertTrue(new_maintenance)
+
+    def test_update_multiple_maintenance_request_record(self):
+        """
+        Test that multiple records of the model 'maintenance.request' can be written simultaneously.
+        """
+        maintenance_requests = self.env['maintenance.request'].create([
+            {
+                'name': 'm_1',
+                'maintenance_type': 'preventive',
+                'kanban_state': 'normal',
+            },
+            {
+                'name': 'm_2',
+                'maintenance_type': 'preventive',
+                'kanban_state': 'normal',
+            },
+        ])
+        maintenance_requests.write({'kanban_state': 'blocked', 'stage_id': self.ref('maintenance.stage_0')})
+        m_2 = maintenance_requests[1]
+        self.assertEqual(m_2.kanban_state, 'blocked')
```

Comment:
I would make the assert on both records
```suggestion
        self.assertRecordValues(maintenance_requests, [
            {'kanban_state': 'blocked', 'stage_id': self.ref('maintenance.stage_0')},
            {'kanban_state': 'blocked', 'stage_id': self.ref('maintenance.stage_0')},
        ])
```